tests/conftest.py

```diff
@@ -1961,6 +1961,7 @@ def spark() -> SparkSession:
 
     spark = (
         SparkSession.builder.appName("PyIceberg integration test")
+        .config("spark.sql.session.timeZone", "UTC")
```

Comment:
Adding this to ensure the integration test results are not affected by local timezone.

I encountered one false-alarm in `test_partitioning_key.py` due to non-UTC local timezone